### DISCUSSION TITLE
Whistleblower identity attachment file

### DISCUSSION THREAD

**Discussion by andreagigliola** [globaleaks/globaleaks-whistleblowing-software#3532] [Q&A] [answered]
Hi everyone,
we have encountered an abnormal behavior when the custodian is active and the following steps are performed:

- I have enabled the custodian role.
- I have overridden the whistleblower_identity template with a custom one, which includes the ability to upload a document as an attachment.
- The whistleblower submits the report, attaching the document.
- The attached document is displayed to the recipient even without the custodian approving the whistleblower's data disclosure request, in the Attachments section.

Can anyone help me understand this?

Thank in advance

Andrea

**Comment by apolastriMki** [comment]
Hi, how do u override whistleblower_identity?
I'm interested on it for modify that fields.Could you help me?
Thanks in advance!
Antonio

**Reply by evilaliv3** [reply]
Dear @apolastriMki , as this is a different question from the original one please open a separated discussion. Thank you

**Comment by evilaliv3** [comment, marked as answer]
Dear @andreagigliola thank you for your question.

This is a know limitation of the current implementation because it was not considered to enable users to upload files in the whistleblower identity question.
We are working on changes that would address this limitation.
